pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -1428,3 +1428,72 @@ ORDER BY id
 1  4004  0
 2  4326  4326
 3  4004  3857
+
+statement ok
+CREATE TABLE geom_linear (
+  dsc  TEXT PRIMARY KEY,
+  geom GEOMETRY
+)
+
+statement ok
+INSERT INTO geom_linear VALUES
+  ('Empty LineString', 'LINESTRING EMPTY'),
+  ('LineString anticlockwise covering all the quadrants', 'LINESTRING(1 -1, 2 2, -2 2, -1 -1)'),
+  ('LineString clockwise covering all the quadrants with SRID 4004', 'SRID=4004;LINESTRING(1 -1, -1 -1, -2 2, 2 2)')
```

Comment:
can you add tests to make sure this behaves as appropriate for POLYGON/POINT/MULTILINESTRING?

Reply:
Since `POLYGON/POINT/MULTILINESTRING` each of them will produce an error, therefore added separately query for them at the bottom. 
Currently, I am unable to figure out how could we add test cases for them by inserting elements in the table.

Similar to this added test case for `POLYGON/POINT/MULTILINESTRING`,
in `unary_operator_test.go` too.